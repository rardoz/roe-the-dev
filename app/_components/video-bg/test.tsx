import { render } from '@testing-library/react'
import VideoBG from '.'

describe('VideoBG', () => {
  it('should render with absolute position', () => {
    expect(render(<VideoBG videoSrc="test.mp4" />).container).toMatchSnapshot()
  })

  it('should render with fixed position', () => {
    expect(
      render(<VideoBG videoSrc="test.mp4" fixed />).container,
    ).toMatchSnapshot()
  })
})
