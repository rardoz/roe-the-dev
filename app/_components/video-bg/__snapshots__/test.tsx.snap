// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoBG should render with absolute position 1`] = `
<div>
  <div
    class="top-0 left-0 w-full h-full -z-10 absolute"
  >
    <video
      autoplay=""
      class=" object-cover w-full h-full"
      loop=""
    >
      <source
        src="test.mp4"
        type="video/mp4"
      />
    </video>
  </div>
</div>
`;

exports[`VideoBG should render with fixed position 1`] = `
<div>
  <div
    class="top-0 left-0 w-full h-full -z-10 fixed"
  >
    <video
      autoplay=""
      class=" object-cover w-full h-full"
      loop=""
    >
      <source
        src="test.mp4"
        type="video/mp4"
      />
    </video>
  </div>
</div>
`;
